### PR TITLE
Write _base.scss 

### PR DESCRIPTION
**Pull request by KamDiaV**

### CODE REVIEW

File: assets/styles/_base.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  box-sizing: border-box;
+}
+
+// html {
+//   font-size: $baseFontSize;
+// }
+
+html,
+body {
+  height: 100%;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: I am not sure that we need this rule.

File: assets/styles/_base.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+p{
+  padding: 0;
+  margin: 0;
+}
+
+image {
+  display: block;
+  min-width: 100%;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: Could you please explain why we need this?

File: assets/styles/_base.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    list-style: none;
+  }
+}
+
+p{
+  padding: 0;
+  margin: 0;
+}
+
+image {
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: Wouldn't it potentially create problems when we use img tag to insert icons?

File: assets/styles/_base.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ul {
+  padding: 0;
+  margin: 0;
+  li {
+    list-style: none;
+  }
+}
+
+p{
+  padding: 0;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: this is already applied by the rule for *

File: assets/styles/_base.scss [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  margin: 0;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: See prev comment

File: assets/styles/_base.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h5,
+h6 {
+  margin-top: 0;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: this is already in force through the rule for *

File: assets/styles/_base.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+  font-weight: normal;
```
Comment (anastasiia-avdeeva) [inline assets/styles/_base.scss]: Not sure why we need it

**Review by anastasiia-avdeeva — CHANGES_REQUESTED** [review]
See comments in the code

**Review by DaryaZhuravleva — APPROVED** [review]

**Review by dianniii — APPROVED** [review]

**Review by anastasiia-avdeeva — APPROVED** [review]

**Pull request merged**